ffmpeg/signcomp_test.go

```diff
@@ -0,0 +1,71 @@
+package ffmpeg
```

Comment:
This file has UNIX permissions set to executable, is that intended?

I also noticed this PR changed `ffmpeg/encoder.c` unix permissions from read/write to read/write/execute `10644 -> 10755`. We should revert that if it was spurious, `chmod -x filename` should do the job I think